app/src/main/java/com/example/searchmovies/view/MovieFragment.kt

```diff
@@ -31,10 +33,21 @@ class MovieFragment : Fragment(R.layout.fragment_movie) {
 
     private fun handleMovies() {
         viewModel.getSearchedMovies().observe(viewLifecycleOwner) { it ->
-            val movieContent = it.movies
-                .joinToString { it.movieContent }
-            view_content_video.text = movieContent
+            view_content_video.text = it.moviesContent
+            view_content_best_video.text =
+                HtmlCompat.fromHtml(
+                    "<b>${it.theBestMovie.title} </b> <br> " +
+                            " ${it.theBestMovie.releaseDate} <br>" +
+                            " ${it.theBestMovie.voteAverage} <br> <br>" +
+                            " ${it.theBestMovie.overview}"
+                    , HtmlCompat.FROM_HTML_MODE_LEGACY
+                )
+            getSearchPoster(it.theBestMovie.posterPath)
         }
     }
 
-}
\ No newline at end of file
+    private fun getSearchPoster(moviePoster: String) {
+        Glide.with(this).load("https://image.tmdb.org/t/p/original/${moviePoster}")
```

Comment:
Budowaniem adresu obrazka powinna zajmować sie inna nazwa niż view. Można to zrobić np. w ViewModelu podczas mapowania obiektu klasy domenowej na `ViewEntity`